src/aura/CRLP_Rollup/CRLP_Rollup.cmp

```diff
@@ -208,16 +208,17 @@
                             <lightning:layoutItem padding="around-medium" smallDeviceSize="6" mediumDeviceSize="3"
                                                   size="12">
                                 <aura:if isTrue="{!not(v.isReadOnly)}">
-                                    <c:CRLP_Tooltip helpText="{!v.labels.createTimeBoundHelp}" fieldLabel="{!v.selectedTimeBoundOperationLabel}"/>
+                                    <c:CRLP_Tooltip helpText="{!v.labels.createDaysHelp}" fieldLabel="{!v.selectedTimeBoundOperationLabel}" isRequired="true"/>
                                 </aura:if>
                                 <lightning:input aura:id="integerInput"
                                                  type="text"
                                                  label="{!v.selectedTimeBoundOperationLabel}"
                                                  value="{!v.activeRollup.intValue}"
                                                  readonly="{!v.isReadOnly}"
-                                                 pattern="^[0-9]*$"
```

Comment:
I noticed that putting in a value of "-1" into the "Days Back" field will generate a validation message, but it will still allow the rollup to be saved.
![image](https://user-images.githubusercontent.com/801017/39760730-1edefd60-52a4-11e8-8421-4301ac4e8da6.png)

Reply:
Good catch, I'll fix this one too!

Reply:
@force2b this has been fixed together with the 0 value issue in the latest commit.